.varci.yml

```diff
@@ -0,0 +1,26 @@
+ruleset:
+  comment_and_close_pull_request:
+    name: "Comment, label & close pull requests"
+    events: [ pull_request ]
+    label: invalid
+    when:
+      - action = "opened"
+    comment: >
+      Hi @{{ user.login }}
+      We have moved to the [forum](http://forum.freecodecamp.com/). Please make your changes there.
+
+      Happy Coding!
+    close: true
+  comment_and_close_issues:
+    name: "Comment, label & close issues"
+    events: [ issues ]
+    label: invalid
+    when:
+      - action = "opened" or action = "reopened"
+      - title matches "/request|fix|create|/i" or body matches "/request|fix|create|/i"
```

Comment:
Do we need to look for specific titles here? Shouldn't all the issues be closed?